src/psyclone/tests/psyir/backend/sir_test.py

```diff
@@ -758,6 +745,24 @@ def test_sirwriter_unary_node_5(parser, sir_writer):
             "with the '-' operator." in str(excinfo.value))
 
 
+# (5/5) Method unaryoperation_node
+def test_sirwriter_unary_node_5(parser, sir_writer):
+    '''Check the unaryoperation_node method of the SIRWriter class outputs
+    the expected SIR when the subject of the unary operator is not a
+    literal.
+
+    '''
+    code = CODE.replace("1.0", "-a(i,j,k)")
```

Comment:
Does this work for an expression too? e.g. `-(a(i,j,k) + b(i,j,k))`. Please could you extend this test.